lib/ansible_tower_client/job_template.rb

```diff
@@ -18,12 +20,25 @@ def survey_spec
       api.get(spec_url).body
     end
 
+    def survey_spec_hash
+      hashify(:survey_spec)
+    end
+
+    def extra_vars_hash
+      hashify(:extra_vars)
+    end
+
     def self.endpoint
       "job_templates".freeze
     end
 
     private
 
+    def self.attr_excluded?(name)
```

Comment:
:disappointed:  - Yeah - I'll pull these out.  I thought I still needed them to allow those attributes to return only a hash.  But since we're trusting the `def launch(options = {})` options hash these are no longer needed